### PR TITLE
{chem}[foss/2023a] MOLGW v3.3

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by lexming** [comment]
@boegelbot please test @ jsc-zen3

File: easybuild/easyconfigs/l/libcint/libcint-5.4.0-gfbf-2023a-pypzpx.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+configopts = "-DWITH_RANGE_COULOMB=on -DWITH_COULOMB_ERF=on -DWITH_F12=on -DENABLE_TEST=on -DPYPZPX=1"
+
+buildopts = 'VERBOSE=1'
+
+runtest = "test "
+separate_build_dir = False  # Must use the same directory for tests
+
+sanity_check_paths = {
+    'files': ['include/cint.h', 'lib/%(name)s.so'],
```
Comment (smoors) [inline easybuild/easyconfigs/l/libcint/libcint-5.4.0-gfbf-2023a-pypzpx.eb:35]:
```suggestion
    'files': ['include/cint.h', 'lib/%%(name)s.%s'], % SHLIB_EXT
```

File: easybuild/easyconfigs/m/MOLGW/MOLGW-3.3-foss-2023a.eb [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+fix_python_shebang_for = ['utils/*.py', 'utils/molgw/*.py']
+
+postinstallcmds = ["cd %(installdir)s/bin && for pyfile in ../utils/*.py; do ln -s $pyfile; done"]
+
+sanity_check_paths = {
+    'files': ['bin/molgw', 'bin/run_molgw.py'],
+    'dirs': ['basis', 'utils']
+}
+
+sanity_check_commands = ["python -c 'import molgw'"]
```
Comment (smoors) [inline easybuild/easyconfigs/m/MOLGW/MOLGW-3.3-foss-2023a.eb:66]:
```suggestion
sanity_check_commands = ["python -s -c 'import molgw'"]
```

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen3

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @lexming!

**Pull request merged**